src/getMoviez.js

```diff
@@ -0,0 +1,156 @@
+import { submitComment, displayComments } from './comments.js';
+import itemCounter from './itemCounter.js';
+
+export const showsList = [];
+const frontmoviez = document.querySelector('.Shows');
+const modalPopUp = document.querySelector('.modal');
+
+const popShow = (arr) => {
+  frontmoviez.innerHTML = '';
+  arr.forEach((movie) => {
+    const eachMovie = `
+    <div class=movie id=${movie.id}>
+      <h2 class="movie-title">${movie.name}</h2>
+      <img class="movie-image" src=${movie.image.medium}>
+      <div class= "userInterAct">
+        <button class="comment-btn" value="${movie.id}">Comment</button>
+        <i class="fas fa-heart" id="${movie.id}"></i>
+      </div>
+    </div>`;
+
+    frontmoviez.insertAdjacentHTML('beforeend', eachMovie);
+  });
+};
+
+function convertDate(date) {
+  const yyyy = date.getFullYear().toString();
+  const mm = (date.getMonth() + 1).toString();
+  const dd = date.getDate().toString();
+  const mmChars = mm.split('');
+  const ddChars = dd.split('');
+  return `${yyyy}-${mmChars[1] ? mm : `0${mmChars[0]}`}-${ddChars[1] ? dd : `0${ddChars[0]}`}`;
+}
+
+export function createModal(showID) {
```

Comment:
- [ ] [OPTIONAL] It is better to use the ES6 features such as `arrow-function` for all your functions, you can learn more [here](https://www.w3schools.com/js/js_arrow_function.asp) about arrow-functions